proto/penumbra/penumbra/view/v1/view.proto

```diff
@@ -689,6 +689,7 @@ message OwnedPositionIdsRequest {
 
 message OwnedPositionIdsResponse {
   core.component.dex.v1.PositionId position_id = 1;
+  core.component.dex.v1.Position position = 2;
 }
```

Comment:
Not convinced that `OwnedPositionIdsResponse` should return `OwnedPositions`; as the method name indicates, it should return `OwnedPositionIds`.